### PR TITLE
Implement gruCell and gru

### PR DESCRIPTION
**Pull request by huningxin**

### CODE REVIEW

File: src/batch_normalization.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import {add, sub, mul, div, pow} from './binary.js';
 import {reshape} from './reshape.js';
-import {Tensor} from './tensor.js';
+import {Tensor, Scalar} from './tensor.js';
```
Comment (BruceDai) [inline src/batch_normalization.js:5]: Thanks for introducing 'Scalar' class.

Please also update https://github.com/huningxin/webnn-baseline/blob/main/test/gemm_test.js#L13, thanks.
```js
        options.c = new Tensor([1], [C]); // L13
```

**Review by BruceDai — COMMENTED** [review]
Thanks @huningxin.
I've updated https://github.com/huningxin/webnn-baseline/pull/11, please rebase it, thanks.

File: src/batch_normalization.js (hunk 1 above)
Comment (huningxin) [reply src/batch_normalization.js:5]: Good catch. Thanks.

Comment (huningxin) [reply src/batch_normalization.js:5]: Fixed it in the new commit. PTAL.

Comment (BruceDai) [reply src/batch_normalization.js:5]: LGTM, thanks.

**Review by BruceDai — APPROVED** [review]

**Pull request merged**